main.py

```diff
@@ -25,7 +30,7 @@ def main():
             time.sleep(SLEEP_INTERVAL)
     except KeyboardInterrupt:
         db_connection.close_db_connection()
-        print("[INFO] end of collect docker stats")
+        logger.info("[INFO] end of collect docker stats")
```

Comment:
Same comments as for necessity of [Err], but here it is for [INFO].

Reply:
resolved